src/components/basePage.tsx

```diff
@@ -79,7 +79,11 @@ export default ({
 
       <div className="row">
         <div
-          className={hasToc ? "col-sm-8 col-md-12 col-lg-8 col-xl-9" : "col-12"}
```

Comment:
this fixed an issue where if there was no toc we were pushing the sdk info to the footer